<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QGroupBox" name="groupLibrary">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Library file</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="pathLibrary"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseLibrary">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupPlugins">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Plugins directory</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="pathPlugins"/>
             </item>
             <item>
              <widget class="QPushButton" name="browsePlugins">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupData">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Data directory</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="pathData"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseData">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupROM">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>ROMs directory</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="pathROM"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseROM">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Emulator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QGroupBox" name="groupGraphics">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
	padding: 4px 0
;
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
           </property>
           <property name="title">
            <string>Graphics</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QCheckBox" name="checkFullscreen">
                <property name="text">
                 <string>Fullscreen</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkOSD">
                <property name="text">
                 <string>On Screen Display</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkEnableVidExt">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable embedding of OpenGL window. This option needs restart.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Enable Video Extension</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupResolution">
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
                </property>
                <property name="title">
                 <string>Resolution</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QComboBox" name="comboResolution">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="toolTip">
                      <string>Used only when video extension is not enabled</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupEmuMode">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex;
	padding: 4px 0;
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
           </property>
           <property name="title">
            <string>CPU Core</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupEmuMode_2">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex;
	padding: 4px 0;
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
           </property>
           <property name="title">
            <string>Compatibility</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <widget class="QCheckBox" name="checkNoCompiledJump">
                <property name="text">
                 <string>No Compiled Jump</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkDisableExtraMem">
                <property name="text">
                 <string>Disable Extra Memory</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>461</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QGroupBox" name="groupVideo">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Video</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="icons.qrc">:/icons/plugins_video.png</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboVideo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonVideo">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Configure</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupAudio">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Audio</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="icons.qrc">:/icons/plugins_audio.png</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboAudio">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonAudio">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Configure</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupInput">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Input</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="icons.qrc">:/icons/plugins_input.png</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboInput">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonInput">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Configure</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupRSP">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex; 
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
          </property>
          <property name="title">
           <string>Rsp</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="icons.qrc">:/icons/plugins_rsp.png</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboRSP">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonRSP">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Configure</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>368</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
